biolink/pieces-parts/rank-regulators.rkt

```diff
@@ -1,8 +1,11 @@
 #lang racket
 (require "query.rkt"
+         "rank-regulators-gene-lists.rkt"
          racket/hash)
 (provide (all-defined-out)
-         (all-from-out "query.rkt" racket/hash))
+         (all-from-out "query.rkt"
+                       "rank-regulators-gene-lists.rkt"
```

Comment:
why this change?

Reply:
I just included that so the user can just load rank-regulators.rkt and not have to load rank-regulators-gene-lists.rkt separately. With my workflow, it was just easier this way.

Reply:
Do you think it might make more sense to have a rank-regulators-lib file which has both rank-regulators and rank-regulators-gene-lists imported, then import that one for the workflow?

I did run into the same workflow need for what it's worth.

Or maybe add a comment that this is not used here, but just more convenient?

Anyways, I think it's OK like that. :)